Readme.md

```diff
@@ -34,6 +34,15 @@ add 20 22
 # 42
 ```
 
+Furthermore, it is possible to use the download & cache infrastructure for arbitrary
```

Comment:
Can we create a new `api.md` file? I would like to expose a few more functions (for example the cache dir logic should be exposed for other scripts to use) in the future so they need a place to be documented.

Reply:
I would be grateful if you could start that document and I would then follow your style to document `import_file`.

Reply:
Added in cfa731d1ca32465a91994c928b0f941bf9ff31e0.

Reply:
Thank you. I will look into it tomorrow.

Should I also keep the `import_file` example in `Readme.md` or would you prefer to have it moved completely to `api.md`?

Reply:
I think just move it to the API page.